src/users/users.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+
+import { CreateUserDto } from '~/users/dto/create-user.dto';
+import { UserModel } from '~/users/model/user.model';
+import { UsersService } from '~/users/users.service';
+
+@Controller('users')
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @ApiOperation({
+    summary: 'Create a new REST API user',
+    description: 'This endpoint creates a new REST API user',
+  })
+  @ApiOkResponse({
+    description: 'The newly created user',
+    type: UserModel,
+  })
+  @Post('/create')
```

Comment:
Haha yeah, its something we want to add.

That said if you exposed the REST API on internal network only and enabled API keys it would still have a reasonable level of security.

Not ready to be exposed to the public internet with signers loaded yet though.